packages/ezresto/ezresto.0.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Nomadic Labs" "Ocamlpro" ]
+license: "LGPL-2.1-with-OCaml-exception"
+homepage: "https://gitlab.com/nomadic-labs/resto"
+bug-reports: "https://gitlab.com/nomadic-labs/resto/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/resto"
+synopsis: "A minimal OCaml library for type-safe HTTP/JSON RPCs"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "dune" "build" "@runtest_ezresto" "-j" jobs ] { with-test }
```

Comment:
```suggestion
  [ "dune" "build" "-p" name "-j" jobs ] { with-test }
```
I think this should work in your case as well. If it does not please use the `--profile=release` argument as the default profile might break packages in the future

Reply:
I tried your suggested changes but it doesn't work. I think you meant `runtest` instead of `build`?
But that also gives some trouble probably because the tests for one package depend on another and it confuses opam/dune.

I'll ship it without the tests in the opam packages. The tests are run on the CI anyway and I'll consider that to be enough for this release.